spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/JGitEnvironmentRepository.java

```diff
@@ -307,6 +341,17 @@ private FetchResult fetch(Git git, String label) {
 			}
 			return result;
 		}
+		catch (TransportException ex){
+			if(errorLogOnFailingFetch){
```

Comment:
The Jgit documentation for the method  fetch says:
> @throws org.eclipse.jgit.api.errors.TransportException 
> when an error occurs during transport

I think that is an error and so we should log it as error